### PR TITLE
Add django-pandas

### PR DESCRIPTION
**Pull request by sannykr**

### CODE REVIEW

**Comment by sannykr** [comment]
@conda-forge-admin : Could you please review and merge this pull request.

File: recipes/django-pandas/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  noarch: python
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - django >=1.8
```
Comment (isuruf) [inline recipes/django-pandas/meta.yaml]: This differs from what's in setup.py

Comment (sannykr) [reply recipes/django-pandas/meta.yaml]: Thanks. I have fixed that

**Comment by sannykr** [comment]
@conda-forge/staged-recipes : appveyor is running forever. Can you please retrigger the appveyor build.

**Comment by scopatz** [comment]
When I clicked on Appveyor it actually said that it ran successfully. merging Thanks!

**Pull request merged**